### PR TITLE
Support for outputfile

### PR DESCRIPTION
**Pull request by crudo**
- [x] documentation
- [x] example
- [x] test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
   },
   "homepage": "https://github.com/wikimedia/grunt-stylelint",
   "dependencies": {
-      "stylelint": "^7.0.0"
+    "chalk": "1.1.3",
+    "stylelint": "7.7.1"
```
Comment (ntwb) [inline package.json]: Why did you remove the caret `^`?

FWIW: This should probably now use the tilde `~` feature for semantic versioning, previously it was the caret `^` but as of stylelint v8.0.0 the recommendation will be the the tilde `~`

**Comment by ntwb** [comment]
I'm not sure about this, `grunt-stylelint` already supports the stylelint custom formatters?

https://github.com/stylelint/stylelint/tree/master/lib/formatters

**Comment by crudo** [comment]
Hi @Volker-E @jdforrester, can we merge this?

**Comment by jdforrester** [comment]
Fine by me (sorry for the rebase necessity; will fix when I get back to it).

**Comment by crudo** [comment]
Hey @jdforrester, it is rebased.
